$NetBSD: patch-docs_CMakeLists.txt,v 1.1 2023/07/07 07:45:51 nia Exp $

Honour CMAKE_INSTALL_MANDIR so different platforms' preference of
man/ vs. share/man/ is respected.

--- docs/CMakeLists.txt.orig	2022-05-31 10:57:53.000000000 +0000
+++ docs/CMakeLists.txt
@@ -5,13 +5,15 @@ set(MAN_PAGES
   mkmamedb.1
 )
 
+include(GNUInstallDirs)
+
 foreach(MAN_PAGE ${MAN_PAGES})
   string(REGEX REPLACE "[1-9]$" "${DOCUMENTATION_FORMAT}" SOURCE_FILE ${MAN_PAGE})
   if(DOCUMENTATION_FORMAT MATCHES "html")
-    install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${MAN_PAGE} DESTINATION share/doc/${PROJECT_NAME} RENAME ${SOURCE_FILE})
+    install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${MAN_PAGE} DESTINATION ${CMAKE_INSTALL_DOCDIR} RENAME ${SOURCE_FILE})
   else()
     string(REGEX REPLACE ".*(.)$" "man\\1" SUBDIR ${MAN_PAGE})
-    install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${MAN_PAGE} DESTINATION man/${SUBDIR})
+    install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${MAN_PAGE} DESTINATION ${CMAKE_INSTALL_MANDIR}/${SUBDIR})
   endif()
   # configure_file does not find out about updates to the sources, and it does not provide a target
   #configure_file(${SOURCE_FILE} ${MAN_PAGE} COPYONLY)
